tests/framework/CodeInterfaceTests/Neutrino/gold/Neutrino_Base/sampleNeutrino/3/NeutrinoInput/Measurements/results.csv

```diff
@@ -0,0 +1,301 @@
+0.0209159,0.017277

```

Comment:
you should remove this... you are not testing against this... you just test _"Neutrino_Base/out_dump.csv"_ in your **tests** file